README.md

```diff
@@ -1420,8 +1420,6 @@ Background sync is not just hard, it is stupid. Here are my technical and philos
 
 ## Known Issues
 
-* It probably doesn't work on Windows.
-* Google Drive allows a directory to contain files/directories with the same name. Client doesn't handle these cases yet. We don't recommend you to use `drive` if you have such files/directories to avoid data loss.
```

Comment:
I'd rather leave this statement in there because drive has to ask folks to fix name clashes(or it can do it automatically). There is also a force mode that just overwrites.

Reply:
I've added it back with a link to learn more.